<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family: 微软雅黑;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>四级词汇</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="audio" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSlider" name="progress">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="time">
           <property name="text">
            <string>--/--</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="speed"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="play">
          <property name="text">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="replay">
          <property name="text">
           <string>重播放</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionnew"/>
    <addaction name="actionpadding_time"/>
    <addaction name="actionrepeat"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>来源</string>
    </property>
    <addaction name="actionbaidu"/>
    <addaction name="actionyoudao"/>
    <addaction name="actionsougou"/>
    <addaction name="actiongoogle"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>代理</string>
    </property>
    <addaction name="actionp_open"/>
    <addaction name="actionp_close"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>播放</string>
    </property>
    <addaction name="actionorder"/>
    <addaction name="actionunorder"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionpadding_time">
   <property name="text">
    <string>intervals</string>
   </property>
  </action>
  <action name="actionnew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionrepeat">
   <property name="text">
    <string>repeat</string>
   </property>
  </action>
  <action name="actionbaidu">
   <property name="text">
    <string>baidu</string>
   </property>
  </action>
  <action name="actionyoudao">
   <property name="text">
    <string>youdao</string>
   </property>
  </action>
  <action name="actionsougou">
   <property name="text">
    <string>sougou</string>
   </property>
  </action>
  <action name="actiongoogle">
   <property name="text">
    <string>google</string>
   </property>
  </action>
  <action name="actionp_open">
   <property name="text">
    <string>p_open</string>
   </property>
  </action>
  <action name="actionp_close">
   <property name="text">
    <string>p_close</string>
   </property>
  </action>
  <action name="actionorder">
   <property name="text">
    <string>order</string>
   </property>
  </action>
  <action name="actionunorder">
   <property name="text">
    <string>unorder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
